lib/kazoo/version.rb

```diff
@@ -1,3 +1,3 @@
 module Kazoo
-  VERSION = "0.5.1"
+  VERSION = "0.5.2"
```

Comment:
FYI: I usually do this as a separate commit directly on master, because I also have to run the release script.